### ISSUE TITLE
Changing microphones 

### ISSUE THREAD

**Issue by Michri** [Vocaluxe/Vocaluxe#367]
Hi, we were playing first time with 4 players AND 4 microphones- cool!
BUT it would be cool, not to change microphones in party modes when amount of players = amount of microphones. It always cause confusion, because we had to shuffle microphones after each round.

Thanks

**Comment by Michri** [comment]
Am I the only one having this issue?

**Comment by KD1975** [comment]
No, and it`s new in Version 0.4 in 0.21 there wasn't the forced mic change

**Comment by daggeg** [comment]
I concur on this one. It would be nice to not have to swap mica inbetween songs if the numbet of players are equal to or less than the number of mics.
